wagtail/contrib/postgres_search/models.py

```diff
@@ -14,6 +14,11 @@
 from .utils import get_descendants_content_types_pks
 
 
+class SearchAutocomplete(SearchQuery):
+    def as_sql(self, compiler, connection):
+        return "to_tsquery(''%s':*')", [self.value]
```

Comment:
Shoudn't this be: ``to_tsquery('''%s'':*')``?

Reply:
No, because psycopg2 escapes the argument, `self.value`. This generates `to_tsquery('''your search'':*')`.